### PR TITLE
Remove bower deps and add bower.json deps on npm

### PR DESCRIPTION
**Pull request by vbardales**
Next step, create a npm package ng-i18next

### CODE REVIEW

**Comment by efolio** [comment]
You should replace the bower_components paths in the Gulpfile for the tests to work properly.

**Comment by vbardales** [comment]
Done :+1:

**Comment by vbardales** [comment]
Perhaps a little too tough to completely remove bower.json :s

**Comment by efolio** [comment]
Why remove the `bower.json`?
Other people that use bower need it…

**Comment by qraynaud** [comment]
Yes, you just need to remove the devDependencies section from it. Otherwise it is good.

**Comment by qraynaud** [comment]
Adding a `.npmignore` file would also be a damn good idea.

File: package.json [hunk 1]
```diff
@@ -16,8 +16,12 @@
     "type": "git",
     "url": "git://github.com/i18next/ng-i18next.git"
   },
+  "dependencies": {
+    "angular": ">=1.2",
+    "i18next-client": "^1.7.4"
```
Comment (qraynaud) [inline package.json]: I believe this should be a `peerDependency` & `devDependency`.

**Comment by vbardales** [comment]
#76 is cleaner (damn, it's hard to work via Github interface only)

**Pull request closed without merging**